ludwig/schema/trainer.py

```diff
@@ -16,175 +17,105 @@
-        min=0.0,
-        max=1.0,
-        description=(
-            "Reduces the learning rate when the algorithm hits a plateau (i.e. the performance on the validation does "
-            "not improve"
-        ),
+        default=0.0, min=0.0, max=1.0, **lsmt.reduce_learning_rate_on_plateau_metadata
     )
 
     reduce_learning_rate_on_plateau_patience: int = schema_utils.NonNegativeInteger(
-        default=5, description="How many epochs have to pass before the learning rate reduces."
+        default=5, **lsmt.reduce_learning_rate_on_plateau_patience_metadata
     )
 
     reduce_learning_rate_on_plateau_rate: float = schema_utils.FloatRange(
-        default=0.5, min=0.0, max=1.0, description="Rate at which we reduce the learning rate."
+        default=0.5, min=0.0, max=1.0, **lsmt.reduce_learning_rate_on_plateau_rate_metadata
     )
 
-    reduce_learning_rate_eval_metric: str = schema_utils.String(default=LOSS, description="")
+    reduce_learning_rate_eval_metric: str = schema_utils.String(
+        default=LOSS, **lsmt.reduce_learning_rate_eval_metric_metadata
+    )
 
-    reduce_learning_rate_eval_split: str = schema_utils.String(default=TRAINING, description="")
+    reduce_learning_rate_eval_split: str = schema_utils.String(
+        default=TRAINING, **lsmt.reduce_learning_rate_eval_split_metadata
+    )
 
     increase_batch_size_on_plateau: int = schema_utils.NonNegativeInteger(
-        default=0, description="Number to increase the batch size by on a plateau."
+        default=0, **lsmt.increase_batch_size_on_plateau_metadata
     )
 
     increase_batch_size_on_plateau_patience: int = schema_utils.NonNegativeInteger(
-        default=5, description="How many epochs to wait for before increasing the batch size."
+        default=5, **lsmt.increase_batch_size_on_plateau_patience_metadata
     )
 
     increase_batch_size_on_plateau_rate: float = schema_utils.NonNegativeFloat(
-        default=2.0, description="Rate at which the batch size increases."
+        default=2.0, **lsmt.increase_batch_size_on_plateau_rate_metadata
     )
 
     increase_batch_size_on_plateau_max: int = schema_utils.PositiveInteger(
-        default=512, description="Maximum size of the batch."
+        default=512, **lsmt.increase_batch_size_on_plateau_max_metadata
     )
 
-    increase_batch_size_eval_metric: str = schema_utils.String(default=LOSS, description="")
+    increase_batch_size_eval_metric: str = schema_utils.String(
+        default=LOSS, **lsmt.increase_batch_size_eval_metric_metadata
+    )
 
-    increase_batch_size_eval_split: str = schema_utils.String(default=TRAINING, description="")
+    increase_batch_size_eval_split: str = schema_utils.String(
+        default=TRAINING, **lsmt.increase_batch_size_eval_split_metadata
+    )
 
-    decay: bool = schema_utils.Boolean(default=False, description="Turn on exponential decay of the learning rate.")
+    decay: bool = schema_utils.Boolean(default=False, **lsmt.decay_metadata)
```

Comment:
I think it's important that we keep the descriptions and defaults visible and readable in this file. Putting everything else in the "metadata" block SGTM.

Reply:
descriptions in particular though can get pretty long, which is my only concern